packages/trader/src/Modules/CFD/Containers/missing-real-account.tsx

```diff
@@ -3,8 +3,9 @@ import React from 'react';
 import { CFD_PLATFORMS } from '@deriv/shared';
```

Comment:
(question) No import for `prop-types` to be removed in this PR?

Reply:
Done